test/manual/mongosh_scopes.test.ts

```diff
@@ -0,0 +1,24 @@
+import { expect } from 'chai';
+
+const expectedMongoshScopes = [
```

Comment:
I think this is an okay approach, we can go with it and see if this becomes too annoying at some point.

I’ll say though, I’d recommend using “package” instead of “scope”. The latter is kind of a lerna-specific term and, in particular, nobody is going to know that it refers to packages if there is no context to clarify it.